src/group_theory/subgroup/basic.lean

```diff
@@ -1865,6 +1868,29 @@ le_antisymm (le_normalizer_comap f)
     simp [hx y]
   end
 
+@[to_additive]
+lemma comap_normalizer_eq_of_injective_range {N : Type*} [group N] (H : subgroup G)
+  {f : N →* G} (hf : function.injective f) (h : H.normalizer ≤ f.range) :
+  H.normalizer.comap f = (H.comap f).normalizer :=
+begin
+  apply (subgroup.map_injective hf),
+  rw map_comap_eq_self h,
+  apply le_antisymm,
+  { refine (le_trans (le_of_eq _) (map_mono (le_normalizer_comap _))),
+    rewrite map_comap_eq_self h, },
+  { refine (le_trans (le_normalizer_map f) (le_of_eq _)),
+    rewrite map_comap_eq_self (le_trans le_normalizer h), }
+end
+
+@[to_additive]
+lemma comap_subtype_normalizer_eq {H N : subgroup G} (h : H.normalizer ≤ N) :
```

Comment:
```suggestion
lemma normalizer_comap_subtype_eq {H N : subgroup G} (h : H.normalizer ≤ N) :
```

Reply:
Unless you meant to swap the LHS and RHS.

Reply:
Ah, makes sense. I guess I originally wrote that lemma without using the dot-notation for the function application.

Reply:
Actually, I checked with `#check`, and lean pretty-prints the theorem as
```
comap N.subtype H.normalizer = (comap N.subtype H).normalizer
```
so I changed the statement to match that, and then the current name seems fine.

Reply:
Personally, I like
`normalizer_comap_subtype_eq : H.normalizer.comap N.subtype = (H.comap N.subtype).normalizer`
because it keeps the dot-notation, and the lemma name sounds right to my ears (if you take the normalizer and then comap along subtype, then this is what you get)

Reply:
I agree that it’s prettier! But as long as lean prints the statements without dot notation (even if you write it with), e.g. in proof states, it seems more useful to keep it that way. This makes it easier for someone faced with such a proof state to find the lemma, I’d say.

If you can somehow get lean to pretty-print it in dot notation as well, I’d agree.